postgresql/endpoint_methods.go

```diff
@@ -0,0 +1,56 @@
+package postgresql
+
+import (
+	"github.com/lib/pq"
+
+	"github.com/benchttp/server/benchttp"
+)
+
+func (c *computedStatsService) ListMetadataByUserID(userID int) ([]*benchttp.Metadata, error) {
+	metadataList := make([]*benchttp.Metadata, 0)
```

Comment:
[suggestion] No need for `make` if not initialized
```suggestion
	metadataList := []benchttp.Metadata{}
```